### DISCUSSION TITLE
200k Concurrent Users

### DISCUSSION THREAD

**Discussion by wildanm** [apostrophecms/apostrophe#3600] [Q&A]
Hi All, 

Is there anyone here who has the experience to deploy apostrophe cms on GCP to serve 150k to 200k concurrent users? 

Thank you.

**Comment by boutell** [comment]
Hello Wildan, sorry for my slow response — we were out for the holidays.

The ApostropheCMS team has experience deploying ApostropheCMS at similar scales, although not specifically on GCP (we have most often used AWS ourselves, and we have worked closely to assist a large in-house client team to do similar work on Azure). I think we would be well-positioned to help you.

If you’d like to connect 1:1 with someone on the Apostrophe core team, please submit a contact form here and we will get the right folks connected with you.

https://apostrophecms.com/contact-us
